ffi/cbindgen.toml

```diff
@@ -0,0 +1,42 @@
+language = "C"
+autogen_warning = "/* Warning, this file is autogenerated by cbindgen. Don't modify this manually. */"
+include_version = true
+cpp_compat = true
+documentation = true
+style = "tag"
+usize_is_size_t = true
+include_guard = "_WIREFILTER_H_"
+
+[enum]
+rename_variants = "QualifiedScreamingSnakeCase"
+
+[export]
+prefix = "wirefilter_"
+renaming_overrides_prefixing = false
+
+[export.rename]
+"Array" = "array"
+"CompilingResult" = "compiling_result"
+"CPrimitiveType" = "primitive_type"
+"CListDefinition" = "list_definition"
+"CType" = "type"
+"DeserializingResult" = "deserializing_result"
+"ExecutionContext" = "execution_context"
+"ExternallyAllocatedByteArr" = "externally_allocated_byte_arr"
+"Filter" = "filter"
+"FilterAst" = "filter_ast"
+"HashingResult" = "hashing_result"
+"Value" = "value"
+"Map" = "map"
+"MatchingResult" = "matching_result"
+"ParsingResult" = "parsing_result"
+"RustAllocatedString" = "rust_allocated_str"
+"Scheme" = "scheme"
+"SerializingResult" = "serializing_result"
+"StaticRustAllocatedString" = "static_rust_allocated_str"
+"Status" = "status"
+"UsingResult" = "using_result"
```

Comment:
Does the `rename_types = "snake_case"` below not take care of this?

Reply:
Unfortunately no, I will remove the export.mangle section as it doesn't appear to do anything